src/sentry.ts

```diff
@@ -32,8 +32,8 @@ export default class SentryService {
 
   initSentry(): void {
     this.ignoreTryCatch(async () => {
-      const sentry_out = process.env.TEST !== '1' && process.env.NODE_ENV != ENVIRONMENT.PREVIEW;
       Sentry.init({
+        enabled: process.env.TEST !== '1' && process.env.NODE_ENV != 'development' && process.env.NODE_ENV != ENVIRONMENT.PREVIEW,
```

Comment:
this will disable sentry using dev command locally for engineers as intended from the debug config.